master/internal/rm/kubernetesrm/pods.go

```diff
@@ -210,8 +215,16 @@ func (p *pods) Receive(ctx *actor.Context) error {
 			}
 		}
 
-		p.startPodInformer(ctx)
-		p.startNodeInformer(ctx)
+		err := p.startPodInformer(ctx)
```

Comment:
In the next iteration of this refactor, the goal would be to remove the `actor.Context` input from `startPodInformer`, `startNodeInformer` etc.